src/analysis/convex/integral.lean

```diff
@@ -31,6 +44,10 @@ variables {α E F : Type*} {m0 : measurable_space α}
   [topological_space.second_countable_topology F] [measurable_space F] [borel_space F]
   {μ : measure α} {s : set E}
 
+/-!
```

Comment:
If you use such a sectioning comment, you should also have one before the strict versions.